<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelFCD</class>
 <widget class="QScrollArea" name="PanelFCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>389</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBoxOpacity">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Thickness
Threshold</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="sliderThreshold">
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditThreshold">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="sliderSigma">
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSigma">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min # of 
Voxels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSlider" name="sliderMinArea">
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMinArea">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetLabels">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderThreshold</sender>
   <signal>sliderReleased()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnSliderThresholdReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSigma</sender>
   <signal>sliderReleased()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnSliderSigmaReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditThreshold</sender>
   <signal>returnPressed()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnTextThresholdReturned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSigma</sender>
   <signal>returnPressed()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnTextSigmaReturned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMinArea</sender>
   <signal>returnPressed()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnTextMinAreaReturned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMinArea</sender>
   <signal>sliderReleased()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnSliderMinAreaReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetLabels</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnLabelSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetLabels</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>PanelFCD</receiver>
   <slot>OnLabelItemChanged(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>659</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSliderThresholdReleased()</slot>
  <slot>OnSliderSigmaReleased()</slot>
  <slot>OnTextThresholdReturned()</slot>
  <slot>OnTextSigmaReturned()</slot>
  <slot>OnSliderMinAreaReleased()</slot>
  <slot>OnTextMinAreaReturned()</slot>
  <slot>OnLabelSelectionChanged()</slot>
  <slot>OnLabelItemChanged(QTreeWidgetItem*)</slot>
 </slots>
</ui>
